### ISSUE TITLE
Allure report unknown

### ISSUE THREAD

**Issue by evgenym87** [fescobar/allure-docker-service#120]
I downloaded container today. Immediately after running the container all is alright.
The command was: 
docker run --user="$(id -u):$(id -g)" -p 5050:5050 -e CHECK_RESULTS_EVERY_SECONDS=3 -e KEEP_HISTORY=1 \
 -v ${PWD}/allure-results:/app/allure-results \
frankescobar/all

--user="$(id -u):$(id -g)" - is because my boss install ubuntu with root for me firstly.

But after a first auto-detect of a new allure results on page of report (/allure-docker-service/projects/default/reports/latest/index.html) - ALLURE REPORT UNKNOWN.

Log of container:
"Detecting results changes for PROJECT_ID: default
Automatic Execution in Progress for PROJECT_ID: default...
Creating history on results directory for PROJECT_ID: default ...
mkdir: cannot create directory ‘/app/allure-docker-api/static/projects/default/results/history’: No such file or directory
Copying history from previous results...
cp: cannot create directory '/app/allure-docker-api/static/projects/default/results/history': No such file or directory
Creating executor.json for PROJECT_ID: default
/app/generateAllureReport.sh: line 57: /app/allure-docker-api/static/projects/default/results/executor.json: No such file or directory
Generating report for PROJECT_ID: default
Report successfully generated to /app/allure-docker-api/static/projects/default/reports/latest
Storing report history for PROJECT_ID: default
BUILD_ORDER:2
Status: 200"

**Comment by fescobar** [comment]
Hi @evgenym87 you need to mount 2 directories:
This one for your allure results files
```
-v ${PWD}/allure-results:/app/allure-results
```
And this one to keep the history reports
```
-v ${PWD}/allure-reports:/app/default-report
```

Are you or any script is removing the allure-results directory before running tests? Because, if the allure-results is deleted docker will lose the reference.

**Comment by evgenym87** [comment]
"Are you or any script is removing the allure-results directory before running tests? Because, if the allure-results is deleted docker will lose the reference."
Because I am running "Checkout" stage of Jenkins, it always delete project before pull changes from Git. So, as I am understand, your service is not for Jenkins.

**Comment by fescobar** [comment]
@evgenym87 you can run it in Jenkins, but you need a different configuration to avoid to mount your directory.
You can use the multiproject configuration and send the results using the API.
https://github.com/fescobar/allure-docker-service#MULTIPLE-PROJECTS---REMOTE-REPORTS
https://github.com/fescobar/allure-docker-service#send-results-through-api

**Comment by fescobar** [comment]
@evgenym87 if you want to use it with Allure Plugin for Jenkins, it's better if you use that plugin. But if you deploy this docker container in any environment you can use it from Jenkins or any other tool.

**Comment by evgenym87** [comment]
I included functions of send_results_jenkins_pipeline.groovy, run buld, and from Jenkins sending success:
[regress_testing] $ /var/lib/jenkins/tools/ru.yandex.qatools.allure.jenkins.tools.AllureCommandlineInstallation/allure/bin/allure generate /var/lib/jenkins/workspace/regress_testing/build/reports/allure-results -c -o /var/lib/jenkins/workspace/regress_testing/allure-report
Report successfully generated to /var/lib/jenkins/workspace/regress_testing/allure-report
Allure report was successfully generated.
Creating artifact for the build.
Artifact was added to the build.
[Pipeline] findFiles
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] readFile
[Pipeline] httpRequest
HttpMethod: POST
URL: http://localhost:5050/allure-docker-service/send-results?project_id=default
Content-Type: application/json
Sending request to url: http://localhost:5050/allure-docker-service/send-results?project_id=default
Response Code: HTTP/1.1 200 OK
Response: 
{"data":{"current_files":["a17adbc6-8d55-45ee-af47-972dc171df68-attachment.png","07a874f7-886c-450e-b41a-842dafd431ba-attachment.txt","27540321-80db-4c6c-9201-634a592664ab-attachment.txt","2d97f27f-d222-4fa9-b9fa-8391185d4e5a-attachment.txt","4b3fbccd-ce83-40d0-b05c-1568eb17881a-result.json","testrun.json","c29b8d59-ba72-47b0-a982-04a206f76a9e-attachment.txt","09393c8d-fe14-4edf-840e-8a51bbac8937-result.json","470cac14-7c93-4043-8fef-942d3ef376de-attachment.txt","fee13810-1a98-4967-a161-41d84ded74de-attachment.txt","34b8e88f-c42a-4d65-ab42-251382a5d9b0-attachment.txt","executor.json","84fbabbe-fb0d-4ff4-9ca3-2ce8a84b6706-attachment.txt","eaef9c94-8ebe-4f1a-9e0c-3e2ce17361d6-attachment.png","35a08308-e2fe-48f7-91c0-e809b7e0421c-result.json","bedf35f5-f434-40a4-ae00-18f72ba64a00-attachment.txt","d861d203-ed9f-4f31-9489-cc8a10c1ad8e-result.json","c75f0f99-f997-4fa6-b55b-6fc080708f32-attachment.txt","2c7c732b-4465-49d8-b29a-cd08c6cbb00f-attachment.txt","1b232a12-d0e3-4b2a-9ca9-71857fd94d37-attachment.txt","3164d00d-4610-43c7-88dd-f035fdbc7d5e-result.json","9c3e801b-345d-48a7-bae6-55d8944ce079-attachment.txt","3ddfe1f3-cc7b-4593-bb35-ce5c600bc221-result.json","4c4542a4-7d67-441e-a149-5060a48afb3b-result.json","fdaa65ed-2d02-4b0e-8774-abd7c601004f-attachment.txt"],"current_files_count":25,"failed_files":[],"failed_files_count":0,"processed_files":["07a874f7-886c-450e-b41a-842dafd431ba-attachment.txt","09393c8d-fe14-4edf-840e-8a51bbac8937-result.json","1b232a12-d0e3-4b2a-9ca9-71857fd94d37-attachment.txt","27540321-80db-4c6c-9201-634a592664ab-attachment.txt","2c7c732b-4465-49d8-b29a-cd08c6cbb00f-attachment.txt","2d97f27f-d222-4fa9-b9fa-8391185d4e5a-attachment.txt","3164d00d-4610-43c7-88dd-f035fdbc7d5e-result.json","34b8e88f-c42a-4d65-ab42-251382a5d9b0-attachment.txt","35a08308-e2fe-48f7-91c0-e809b7e0421c-result.json","3ddfe1f3-cc7b-4593-bb35-ce5c600bc221-result.json","470cac14-7c93-4043-8fef-942d3ef376de-attachment.txt","4b3fbccd-ce83-40d0-b05c-1568eb17881a-result.json","4c4542a4-7d67-441e-a149-5060a48afb3b-result.json","84fbabbe-fb0d-4ff4-9ca3-2ce8a84b6706-attachment.txt","9c3e801b-345d-48a7-bae6-55d8944ce079-attachment.txt","a17adbc6-8d55-45ee-af47-972dc171df68-attachment.png","bedf35f5-f434-40a4-ae00-18f72ba64a00-attachment.txt","c29b8d59-ba72-47b0-a982-04a206f76a9e-attachment.txt","c75f0f99-f997-4fa6-b55b-6fc080708f32-attachment.txt","d861d203-ed9f-4f31-9489-cc8a10c1ad8e-result.json","eaef9c94-8ebe-4f1a-9e0c-3e2ce17361d6-attachment.png","executor.json","fdaa65ed-2d02-4b0e-8774-abd7c601004f-attachment.txt","fee13810-1a98-4967-a161-41d84ded74de-attachment.txt","testrun.json"],"processed_files_count":25,"sent_files_count":25},"meta_data":{"message":"Results successfully sent for project_id 'default'"}}

Success code from [200‥200]

But when I go to host:5050/allure-docker-service/projects/default/reports/latest/index.html, I see ALLURE REPORT UNKNOWN
Container log:
 Not checking results automatically
 ALLURE_VERSION: 2.13.5
 Generating default report
 ls: cannot access '/app/allure-docker-api/static/projects/default/reports': No such file or directory
 Creating results directory for PROJECT_ID: default
 Creating executor.json for PROJECT_ID: default
 Generating report for PROJECT_ID: default
 Report successfully generated to /app/allure-docker-api/static/projects/default/reports/latest
 Status: 200
 2020-09-02 14:38:44.001:INFO::main: Logging initialized @382ms to org.eclipse.jetty.util.log.StdErrLog

Before actions above, I launched allure-service by command:
docker run -p 5050:5050 -e CHECK_RESULTS_EVERY_SECONDS=NONE -e KEEP_HISTORY=1 \
                 -v ${PWD}/projects:/app/projects \
                 frankescobar/allure-docker-service

What I doing wrong?
